File: app/src/main/java/com/digicoffer/lauditor/TimeSheets/Models/TSMatterModel.java
package com.digicoffer.lauditor.TimeSheets.Models;

import org.json.JSONArray;

public class TSMatterModel {
    String Mattername;
    String Matterid;
    String Matter_type;
    Boolean iseditable = true;
    JSONArray Tasks;

    public Boolean getIseditable() {
        return iseditable;
    }

    public void setIseditable(Boolean iseditable) {
        this.iseditable = iseditable;
    }

    public JSONArray getTasks() {
        return Tasks;
    }

    public void setTasks(JSONArray tasks) {
        Tasks = tasks;
    }

    public String getMatter_type() {
        return Matter_type;
    }

    public void setMatter_type(String matter_type) {
        Matter_type = matter_type;
    }

    public String getMattername() {
        return Mattername;
    }

    public void setMattername(String mattername) {
        Mattername = mattername;
    }

    public String getMatterid() {
        return Matterid;
    }

    public void setMatterid(String matterid) {
        Matterid = matterid;
    }
}

File: app/src/main/java/com/digicoffer/lauditor/TimeSheets/Models/TSMatterModel.kt
package com.digicoffer.lauditor.TimeSheets.Models

import org.json.JSONArray

class TSMatterModel {
    @get:JvmName("getMattername")
    @set:JvmName("setMattername")
    var mattername: String? = null

    @get:JvmName("getMatterid")
    @set:JvmName("setMatterid")
    var matterid: String? = null

    @get:JvmName("getMatter_type")
    @set:JvmName("setMatter_type")
    var matter_type: String? = null
    
    @get:JvmName("getIseditable")
    @set:JvmName("setIseditable")
    var iseditable: Boolean? = true
    
    @get:JvmName("getTasks")
    @set:JvmName("setTasks")
    var Tasks: JSONArray? = null
}
